core/server/api_container/server/startosis_engine/kurtosis_types/service_config/service_config.go

```diff
@@ -124,9 +128,13 @@ func NewServiceConfigType() *kurtosis_type_constructor.KurtosisTypeConstructor {
 					Validator: func(value starlark.Value) *startosis_errors.InterpretationError {
 						return builtin_argument.Uint64InRange(value, MemoryAllocationAttr, minimumMemoryAllocationMegabytes, math.MaxUint64)
 					},
-					Deprecation: starlark_warning.Deprecation(starlark_warning.DeprecationDate{
-						Day: 25, Year: 2023, Month: 6, //nolint:gomnd
-					}, "This field is being deprecated in favour of `max_memory` to set maximum memory a container can use"),
+					Deprecation: starlark_warning.Deprecation(
+						starlark_warning.DeprecationDate{
+							Day: 25, Year: 2023, Month: 6, //nolint:gomnd
```

Comment:
Should we remove this? 25/06 is like 3 months ago...

Reply:
I'm reviewing this with PK, it seems our code is still depending on these arguments

Reply:
I created this ticket https://github.com/kurtosis-tech/kurtosis/issues/1324